substrate-node-example/runtime/src/lib.rs

```diff
@@ -9,7 +9,7 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-// use orml_nft;
+use orml_nft;
```

Comment:
what exactly this PR do? no difference from branch main if merge main into this branch

Reply:
strange... I added the orml_nft configs into runtime. But seems already merged into main...

Reply:
emmm, I fixed and merged that yesterday =.=